cfg/latest_board.json

```diff
@@ -0,0 +1,44 @@
+{
+  "type": "SYS_SINGLE",
+  "description": "Test board with random detectors info",
+  "id": 12345678,
+  "detectors": [
+    {
+      "type": "det",
+      "mode": 3,
+      "channel": 0,
+      "coordinates": {
+        "default": [20, 25],
+        "calibrated": [
+          {
+            "command": "halign",
+            "z": 5,
+            "data": {
+              "coorodinates": [21, 24],
```

Comment:
typo: `coordinates`

Reply:
Also note that as per halign commands, we are storing `x`, `x_fit_uncertainty`, `y`, `y_fit_uncertainty` here. So place 4 numbers for format testing.